docs/api-docs/storefront/graphql/graphql-site-content.mdx

```diff
@@ -0,0 +1,1004 @@
+        id
+        ... on WebPage {
+          entityId
+          parentEntityId
+          name
+          isVisibleInNavigation
+          seo {
+            metaKeywords
+            metaDescription
+          }
+        }
+        ... on NormalPage {
+		  path
+          plainTextSummary (characterLimit: 100)
+        }
+      }
+    }
+  }
+}
+
+```
+
+</Tab>
+<Tab>
+
+```json filename="Example query: Get plain-text page" showLineNumbers copy
+{
+  "data": {
+    "site": {
+      "route": {
+        "node": {
+          "id": "Tm9ybWFsUGFnZToy",
+          "entityId": 2,
+          "parentEntityId": null,
+          "name": "Shipping & Returns",
+          "isVisibleInNavigation": true,
+          "seo": {
+            "metaKeywords": "Shipping and Return Policy",
+            "metaDescription": "Shipping and Return Policy"
+          },
+          "path": "/shipping-returns/",
+          "plainTextSummary": "To edit this page, log in to your control panel and go to Storefront › Web Pages. Click Edit next..."
+        }
+      }
+    }
+  }
+}
+```
+
+</Tab>
+</Tabs>
+
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```graphql filename="Example query: Get contact form page" showLineNumbers copy
+query {
+  site {
+    route(path: "/contact-us-form/") {
+      node {
+        id
+        ... on WebPage {
+          entityId
+          parentEntityId
+          name
+          isVisibleInNavigation
+          seo {
+            metaKeywords
+            metaDescription
+          }
+        }
+        ... on ContactPage {
+		  path
+          plainTextSummary (characterLimit: 100)
+		  contactFields
+        }
+      }
+    }
+  }
+}
+```
+
+</Tab>
+<Tab>
+
+```json filename="Example query: Get contact form page" showLineNumbers copy
+{
+  "data": {
+    "site": {
+      "route": {
+        "node": {
+          "id": "Q29udGFjdFBhZ2U6Ng==",
+          "entityId": 6,
+          "parentEntityId": null,
+          "name": "Contact Us Form",
+          "isVisibleInNavigation": true,
+          "seo": {
+            "metaKeywords": "Contact Us",
+            "metaDescription": "Contact Info"
+          },
+          "path": "/contact-us-form/",
+          "plainTextSummary": "Visitors can use the form on this page to submit questions or feedback.&nbsp;\r\nStore is located in...",
+          "contactFields": [
+            "fullname",
+            "phone"
+          ]
+        }
+      }
+    }
+  }
+}
+```
+
+</Tab>
+</Tabs>
+
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```graphql filename="Example query: Get raw HTML page" showLineNumbers copy
+query {
+  site {
+    route(path: "/random-page/") {
+      node {
+        id
+        ... on WebPage {
+          entityId
+          parentEntityId
+          name
+          isVisibleInNavigation
+          seo {
+            metaKeywords
+            metaDescription
+          }
+        }
+        ... on RawHtmlPage {
+		  path
+          plainTextSummary (characterLimit: 100)
```

Comment:
Good point, I removed plainTextSummary and added htmlBody for the raw pages 

@jmwiese Tagging you here for visibility (I think the field currently exists on the type because I see it in the Playground Schema)